### PR TITLE
F wrapper and bugfix

### PR DESCRIPTION
**Pull request by dellaert**
- ~changed internal representation of F to fix a bug when flipping U or V~
- now flip columns of U and V which always allows for $U*diag(1,s,0)*V^T$
- added wrapper definitions
- added python unit test

### CODE REVIEW

**Comment by dellaert** [comment]
@akshay-krishnan , @travisdriver  please look at FundamentalMatrix.h first, then rest will make sense. This PR should also make the last PR easier to review.

File: gtsam/geometry/FundamentalMatrix.h [hunk 1]
```diff
@@ -15,34 +15,36 @@ namespace gtsam {
...
-   *
-   * @param U Left rotation matrix
-   * @param s Scalar parameter for the fundamental matrix
-   * @param V Right rotation matrix
+   * Initializes the FundamentalMatrix From the SVD representation
+   * U*diag(1,s,0)*V^T. It will internally convert to using SO(3).
    */
-  FundamentalMatrix(const Rot3& U, double s, const Rot3& V)
-      : U_(U), s_(s), V_(V) {}
+  FundamentalMatrix(const Matrix& U, double s, const Matrix& V);
```
Comment (dellaert) [inline gtsam/geometry/FundamentalMatrix.h]: Should be Matrix3

File: gtsam/geometry/FundamentalMatrix.h [hunk 2]
```diff
@@ -54,22 +56,33 @@ class GTSAM_EXPORT FundamentalMatrix {
    */
   FundamentalMatrix(const Matrix3& F);
 
+  /**
+   * @brief Construct from essential matrix and calibration matrices
+   *
+   * Initializes the FundamentalMatrix from the given essential matrix E
+   * and calibration matrices Ka and Kb.
```
Comment (dellaert) [inline gtsam/geometry/FundamentalMatrix.h]: Provide equation

File: gtsam/geometry/FundamentalMatrix.cpp [hunk 3]
```diff
@@ -47,28 +52,44 @@ FundamentalMatrix::FundamentalMatrix(const Matrix3& F) {
...
+  s_ = s;
+  sign_ = 1.0;
 
-  // Check if V is a reflection
-  if (V.determinant() < 0) {
-    V = -V;
-    s_ = -s_;  // Change sign of scalar if U is a reflection
+  // Check if U is a reflection and its determinant is close to -1 or 1
+  double detU = U.determinant();
+  if (std::abs(std::abs(detU) - 1.0) > 1e-9) {
```
Comment (dellaert) [inline gtsam/geometry/FundamentalMatrix.cpp]: This is always true so I’ll kill it

File: gtsam/geometry/FundamentalMatrix.h (hunk 2 above)
Comment (dellaert) [reply gtsam/geometry/FundamentalMatrix.h]: Done

File: gtsam/geometry/FundamentalMatrix.cpp (hunk 3 above)
Comment (dellaert) [reply gtsam/geometry/FundamentalMatrix.cpp]: Done

**Review by akshay-krishnan — APPROVED** [review]
I did not look at the tests, rest LGTM!

**Pull request merged**